src/Core/Services/MetadataProviders/ISqlMetadataProvider.cs

```diff
@@ -197,5 +197,7 @@ public DatabaseObject GetDatabaseObjectForGraphQLType(string graphqlType)
         /// mode, it returns false.
         /// </summary>
         public bool IsDevelopmentMode();
+
+        public ForeignKeyDefinition GetFKDefinition(string sourceEntityName, string targetEntityName, string referencedEntityName, string referencingEntityName) => throw new NotImplementedException();
```

Comment:
nit: order of params would be more cohesive if source/target entityName order matched referenced/referencing order:

source
target
referencing
referenced

Reply:
and need function comment.